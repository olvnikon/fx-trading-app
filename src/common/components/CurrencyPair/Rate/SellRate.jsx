// @flow
import React from 'react';
import Rate from './Rate';
import styles from './Rate.scss';

type Props = {|
  +currency: string,
  +rate: number,
|};

const SellRate = ({ currency, ...rest }: Props) => (
  <div className={styles.sellRate}>
    <Rate header={`Sell ${currency}`} {...rest} />
  </div>
);

export default SellRate;
