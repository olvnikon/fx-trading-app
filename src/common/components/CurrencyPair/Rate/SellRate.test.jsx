// @flow
import React from 'react';
import { shallow } from 'enzyme';
import SellRate from './SellRate';

describe('SellRate', () => {
  const RATE = 12.345;
  const CURRENCY = 'USD';

  test('should match snapshot', () => {
    const props = {
      rate: RATE,
      currency: CURRENCY,
    };
    const component = shallow(<SellRate {...props} />);
    expect(component).toMatchSnapshot();
  });
});
