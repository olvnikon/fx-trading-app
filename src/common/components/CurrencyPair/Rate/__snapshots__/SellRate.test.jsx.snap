// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SellRate should match snapshot 1`] = `
<div>
  <Rate
    header="Sell USD"
    rate={12.345}
  />
</div>
`;
